### ISSUE TITLE
automatic refresh to get strikethrough / update colors

### ISSUE THREAD

**Issue by finnove** [usernamehw/vscode-todo-md#86]
- When pressing alt-d to complete a task I do not immediately get a refresh for the strike through formatting
- When moving tasks with alt-up/down the colors will be off. E.g. green project will become white. the blue color for the tag will color the whole line or similar.

Workaround:
- toggle sort of tags will make a refresh of the page, and colors will be ok again. I don't know what else will force a refresh. 

Expected behaviour:
- I would expect this to work automatically without any manual update

**Comment by usernamehw** [comment]
Is this issue about editor or webview? In any case, can't reproduce:

![Demo](https://github.com/usernamehw/vscode-todo-md/assets/9638156/e4d421f9-ae2d-4f95-bfd9-abd5ae1f76f4)

**Comment by finnove** [comment]
![image](https://github.com/usernamehw/vscode-todo-md/assets/1630604/a33a5f4f-02e8-4fc9-81ec-151e0402a5f6)
Cool. That inspired me to reinstall the plugin. Another markdown plugin was told to reload, so I uninstalled that as well as a test. Then installed again. Now it works! 👏 But I have another icon than you. (btw I tried 4-5 of these todo plugins before I settled on this, dunno if that is relevant. they are uninstalled now, at least it looks so). Thanks!
